fastlane_bot/helpers/wrap_unwrap_processor.py

```diff
@@ -1,340 +1,104 @@
-from typing import Any, List, Dict
-
-from fastlane_bot.helpers import TradeInstruction, RouteStruct
 from dataclasses import asdict
+from fastlane_bot.helpers import TradeInstruction, RouteStruct
 from fastlane_bot.events.exceptions import FlashloanTokenException
 
-
-class WrapUnwrapProcessor:
+def add_wrap_or_unwrap_trades_to_route(
+    cfg,
```

Comment:
why not include type-hints for the `cfg` argument?

Reply:
Fixed